submissions/israelOladosu/php-db/index.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+include('./classes/queryDb.class.php');
+
+    // an instance of queryDb class to call the getAllUsers method;
+    $queryDb = new QueryDb();
+
+    $results = isset($_COOKIE['users'])? unserialize($_COOKIE['users']) : $queryDb->getAllUsers();
+
+    if(count($results) > 0) {
```

Comment:
Well done @Israel-Force But I wish you are not echoing all the time. There is a better way to do this